### DISCUSSION TITLE
It's like you read my mind

### DISCUSSION THREAD

**Discussion by elsplatto** [tailwindlabs/tailwindcss#4917] [Kind words]
I only get to do a bit of coding from time to time nowadays but thought I would search for the best CSS framework. This Tailwind CSS thing kept on topping the list - and now I see why. Thank you so much for putting into a framework what I was doing by hand previously. Love the utility first approach. My custom CSS is about 5 lines - and even then I suspect it's because I am not as fluent with Tailwind as I soon will be. Amazing repo!

**Comment by simonswiss** [comment]
> Thank you so much for putting into a framework what I was doing by hand previously.

I can relate to this – I think many of us started having a few utilities for font sizes, max-width, etc...

Thank you for the kind words, glad to hear you're enjoying working with Tailwind CSS ✨
